recipes/sfepy/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set rev = "release_2016.3.2" %}
+{% set name = "sfepy" %}
+{% set org = "sfepy" %}
+{% set version = "2016.3.2" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/{{ org }}/{{ name }}.git
+  git_rev: {{ rev }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - python
+    - numpy
+    - scipy
+    - cython
+    - sympy
+    - pytables
+    - mpi4py
+    - matplotlib
+
+  run:
+    - python
+    - numpy
+    - scipy
+    - pyparsing
+    - sympy
+    - cython
```

Comment:
@synapticarbors Thanks for looking at the recipe. I think this would do:

```
requirements:
  build:
    - python
    - numpy
    - cython

  run:
    - python
    - numpy
    - scipy
    - pyparsing
    - sympy
    - pytables
    - mpi4py
    - matplotlib
```

@wd15 Could you try the above changes?

As for skipping the windows build, we did not find how to disable only Windows x86 with Python 3.5 (IIRC), see above: "it seems that pytables is unavailable for Python 3.5 on Windows x86," - this configuration failed, saying that requirements for pytables could not be satisfied, while other windows configurations (e.g. Python 3.4) were OK.